uint8 number_points
multiwii/Waypoint[] waypoints